server/cmwell-data-tools-app/src/main/resources/logback.xml

```diff
@@ -107,15 +107,14 @@
 
     <appender name="ASYNC_BAD_DATA" class="ch.qos.logback.classic.AsyncAppender">
         <appender-ref ref="BAD_DATA" />
-        <maxFlushTime>0</maxFlushTime>
         <discardingThreshold>0</discardingThreshold>
     </appender>
 
     <logger name="tools-red-logger" additivity="true">
         <appender-ref ref="ASYNC_RED_LOG" />
     </logger>
 
-    <logger name="bad-data" additivity="false">
+    <logger name="bad-data" additivity="true">
```

Comment:
Do you want the bad-data to clutter the main log file? This is the current state...
The additivity should be false or am I missing something (maybe I am out of context and the log lines should be printed both on the main log file and also on the bad data one).

Reply:
My mistake. Good spot. I was experimenting...